docs/docs/config.md

```diff
@@ -0,0 +1,54 @@
+For multiple modules with common elements we can define a configuration to apply to all modules, or all modules of a certain type. Maintaining a config helps reduce the code repitition and allows you to change settings accross many widgets in one place. A config object can be passed as third, optional argument to `initializeWidgets`.
```

Comment:
Agree, going to work on this in the next commit, it can have all the top level methods and link out to these reference pages to go in depth.